src/howe_lab_to_nwb/vu2024/metadata/vu2024_fiber_photometry_metadata.yaml

```diff
@@ -170,4 +179,11 @@ Ophys:
       - name: tdTomato
         description: Red static marker
         label: pAAV-CAG-tdTomato(codon diversified-AAV1)
-
+    Photodetectors:
+      - name: CMOSCamera
+        description: A tube lens in each path (Thor labs, No TTL165-A, detected wavelength bandwidth 350-700 nm) focused emission light onto the CMOS sensors of the cameras to form an image of the fiber bundle.
+        manufacturer: Hamamatsu Photonics
+        model: Orca Fusion BT Gen III
+        detector_type: camera
+        # not sure where to get this https://www.hamamatsu.com/content/dam/hamamatsu-photonics/sites/documents/99_SALES_LIBRARY/sys/SCAS0143E_C15440-20UP.pdf
+        detected_wavelength_in_nm: 525.0
```

Comment:
@alessandratrapani not sure this is the correct value, didn't find info on this.